build/e2e-image/entrypoint.sh

```diff
@@ -31,5 +31,5 @@ echo "Waiting consul port-forward to launch on 8500..."
 timeout 60 bash -c 'until printf "" 2>>/dev/null >>/dev/tcp/$0/$1; do sleep 1; done' 127.0.0.1 8500
 echo "consul port-forward launched. Starting e2e tests..."
 consul lock -child-exit-code=true -timeout 30m -try 30m -verbose LockE2E /root/e2e.sh
-killall -q kubectl
-
+killall -q kubectl || true
```

Comment:
@Kuqd for interest's sake - this was the fix for the e2e tests. even with `-q` this was always returning a non zero status code. :man_shrugging: